### PR TITLE
prefix url with scheme

### PR DESCRIPTION
**Pull request by durandom**
otherwise it'll result in a broken link

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -53,7 +53,7 @@ workinggroups:
     time: "10:00"
     tz: ET (East Time)
     frequency: weekly
-    url: meet.google.com/qkt-yacp-wzm
+    url: http://meet.google.com/qkt-yacp-wzm
```
Comment (goern) [inline sigs.yaml:56]: is it really http? not https?

**Review by goern — CHANGES_REQUESTED** [review]

File: sigs.yaml (hunk 1 above)
Comment (durandom) [inline sigs.yaml:56]:
```suggestion
    url: https://meet.google.com/qkt-yacp-wzm
```

File: sigs.yaml [hunk 2]
```diff
@@ -77,7 +77,7 @@ workinggroups:
     time: "11:00"
     tz: ET (East Time)
     frequency: weekly
-    url: meet.google.com/zsu-avba-enx
+    url: http://meet.google.com/zsu-avba-enx
```
Comment (durandom) [inline sigs.yaml:80]:
```suggestion
    url: https://meet.google.com/zsu-avba-enx
```

**Pull request merged**